pixi.toml

```diff
@@ -43,10 +43,18 @@ hypothesis = "*"
 onnxruntime = "*"
 mypy = ">=1.10.0,<1.11"
 ndindex = "*"
+
 [feature.test.tasks]
 test = "pytest"
 test-coverage = "pytest --cov=ndonnx --cov-report=xml --cov-report=term-missing"
-arrayapitests = "ARRAY_API_TESTS_MODULE=ndonnx pytest --max-examples 2 api-coverage-tests/array_api_tests/ --json-report --json-report-file=api-coverage-tests.json -n auto -vv --skips-file=skips.txt"
+
+[feature.test.tasks.arrayapitests]
+cmd = "pytest --max-examples 100 api-coverage-tests/array_api_tests/  -v -rfX --json-report --json-report-file=api-coverage-tests.json -n auto --disable-deadline --disable-extension linalg --skips-file=skips.txt --xfails-file=xfails.txt --hypothesis-seed=0"
```

Comment:
Note the seed! This makes things much easier for CI, but we miss out on the "fuzzing" we would otherwise get over time.

Reply:
Turns out the seed didn't save us from flakiness after all...